src/test/java/com/salesforce/dynamodbv2/BatchGetTest.java

```diff
@@ -39,7 +38,8 @@
 class BatchGetTest {
 
     @ParameterizedTest(name = "{arguments}")
-    @ArgumentsSource(DefaultArgumentProviderForTable1.class)
+    @ArgumentsSource(DefaultArgumentProvider.class)
+    @DefaultArgumentProviderConfig(tables = {TABLE1})
```

Comment:
Introduced a new annotation for tests using DefaultArgumentProvider, so we can specify which test tables to set up in a nicer way, and also so that we can specify Dynalite to be used instead of DynamoDB Local in select test cases.